src/app-root.ts

```diff
@@ -395,6 +395,55 @@ export class AppRoot extends LitElement {
               </label>
             </div>
           </fieldset>
+          <fieldset class="Placeholder-type">
+            <legend>Placeholder type</legend>
+            <div class="checkbox-control">
+              <input
```

Comment:
I would make them radiobutton group which match with other checkbox styling :)

Reply:
I tried to even do radio buttons but they wouldn't toggle, so I figured this is the easiest way for me to get something on the board that works. 👍 I will hold as buttons for now as I am closer to main behavioural fix, this demo helped me come up with creating dynamic interstitial...